scss/pager/_theme.scss

```diff
@@ -54,12 +58,13 @@
         .k-state-selected:hover {
             border-width: 1px;
             border-style: solid;
-            @include fill( $selected-text, $selected-bg, $selected-border, none );
+            background-color: $pager-selected-bg;
+            border-color: $pager-selected-border;
             z-index: 2;
             cursor: default;
 
             &.k-link {
-                color: $selected-text;
+                color: $pager-selected-text;
```

Comment:
Nope, both - **k-link** and **k-state-selected** classes are on the same element